prac_05/emails.py

```diff
@@ -0,0 +1,25 @@
+email_to_name = {}
+
+while True:
+    user_email = input("Email: ")
+    if user_email == "":
+        break
+
+    name = " ".join(user_email.split("@")[0].split(".")).title()
+
+    yes_list = ["yes", "ye", "y", "1", ""]
+    no_list = ["no", "n"]
+    user_name_correct = input(f"Is your name {name}? (Y/N)").lower()
+
+    if (user_name_correct in yes_list):
+        pass
+    elif user_name_correct in no_list:
```

Comment:
Could you optimise this to not have an empty if statement? Perhaps use the "not in" operator.